data_summaries/Bactrian-X.json

```diff
@@ -0,0 +1,320 @@
+{
+    "bactrian-x-en": {
+        "Unique Dataset Identifier": "bactrian-x-en",
+        "Dataset Name": "Bactrian-X (English)",
+        "Paper Title": "Bactrian-X: Multilingual Replicable Instruction-Following Models with Low-Rank Adaptation",
+        "Dataset URL": "https://huggingface.co/datasets/MBZUAI/Bactrian-X",
+        "GitHub URL": "https://github.com/mbzuai-nlp/bactrian-x",
+        "Hugging Face URL": "https://huggingface.co/datasets/MBZUAI/Bactrian-X",
+        "ArXiv URL": "https://arxiv.org/abs/2305.15011",
+        "Semantic Scholar Corpus ID": 258865265,
+        "Collection": "Bactrian-X",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Question Answering",
+            "Summarization",
+            "Open-Domain Relation Extraction",
+            "Text Classification"
+        ],
+        "Text Sources": [
```

Comment:
I would say put this as "crowdsourced" since dolly was originally crowdsourced.

Then move these 3 models to "Model Generated" (which asks whether models were used to generate the text), as well as ChatGPT which was used to get natural responses in these languages.